build.gradle.kts

```diff
@@ -79,7 +79,27 @@ apiValidation {
    * Classes (fully qualified) that are excluded from public API dumps even if they
    * contain public API.
    */
-//  ignoredClasses.add("com.company.BuildConfig")
+  ignoredClasses.addAll(
```

Comment:
just a question
can use smth like `com.mapbox.maps.*.BuildConfig`

Reply:
sadly no, it only takes fully qualified class names, e.g. the full path of the class without wildcards.

Reply:
Yes, would be good to have an internal doc on how to create new plugins/modules to the system, such a guide/readme does not exist yet, will ticket it out to follow up.